integration-libs/opf/base/root/services/opf-resource-loader.service.ts

```diff
@@ -87,7 +87,10 @@ export class OpfResourceLoaderService {
     keyValueList?.forEach((keyValue: OpfKeyValueMap) => {
       attributes[keyValue.key] = keyValue.value;
     });
-    if (attributes?.[this.OPF_RESOURCE_LOAD_ONCE_ATTRIBUTE_KEY] === 'true') {
```

Comment:
it was the opposite logic, we need:
if OPF_RESOURCE_LOAD_ONCE_ATTRIBUTE_KEY  does not have 'true value' -> we add OPF_RESOURCE_ATTRIBUTE_KEY attribute